memory/memtester.py.data/memtester.yaml

```diff
@@ -0,0 +1,5 @@
+test_memster:
+    memtest_opt:
+        runs: 1 
+        memory: null
```

Comment:
i think when memory is `null` memtester won't run 

```
root@localhost memtester]# ./memtester null 1
memtester version 4.3.0 (64-bit)
Copyright (C) 2001-2012 Charles Cazabon.
Licensed under the GNU General Public License version 2 (only).

pagesize is 65536
pagesizemask is 0xffffffffffff0000

Usage: ./memtester [-p physaddrbase [-d device]] <mem>[B|K|M|G] [loops]
[root@localhost memtester]# 

```
getting this can you pass some valid memory value like 100K